tests/js/client/ldapfailover/auth-dualldap.js

```diff
@@ -0,0 +1,236 @@
+/*jshint globalstrict:false, strict:false */
```

Comment:
Cann't we get the three test files into a single one?
They do test exactly the same scenarios, just with different backend configuration.
Depending on the backend the result should slightly differ.

While the tests themselves are correct, it is super hard to reason about them and it is even harder to add new tests / modify something here, as you need to do it in 3 almost identical places.

I am also happy with a "module" that one can call with the Ldap configuration from this files (e.g. {primary: true/false, secondary: true/false}) and then have the 3 test files just call this module with their specific configurations, but that we have the one test-suite in a single file only.

Reply:
updated as suggested